crawl-ref/source/travel.cc

```diff
@@ -3220,7 +3230,7 @@ static bool _find_transtravel_square(const level_pos &target, bool verbose)
             if (!maybe_traversable)
                 mpr("Sorry, I don't know how to traverse that place.");
             else
-                mpr("Sorry, I don't know how to get there.");
+                mprf("Sorry, I don't know how to get there%s", (fail_reasons[current] == "" ? "." : ": " + fail_reasons[current]).c_str());
```

Comment:
Great question! My memory is kind of foggy on this context so I honestly can't remember for sure what the reason is, but I think it might have been one of two things:

1) I simply wasn't sure what the best UX would have been for formatting multiple exclusions.

2) I believe I remember in testing finding that in certain cases autotravel will fail and immediately inform the player, which I expected, but in others I was surprised to see that it does actually travel a few floors and only halts and informs the player when the failure reason is on the floor they've arrived at. I may have gone with the map to try to give the player the most relevant information for those cases.